prov/efa/src/efa_base_ep.h

```diff
@@ -12,6 +12,13 @@
 #include "ofi_util.h"
 #include "rdm/efa_rdm_protocol.h"
 
+#if HAVE_EFADV_SL
+enum efa_qp_service_level {
+    SL_0 = 0,
+    SL_8 = 8,
+};
```

Comment:
This is pretty superfluous unless `SL_n` can ever be anything other than `n`. We should just define a macro that represents whatever default value we want to use. E.g. `EFA_QP_DEFAULT_SERVICE_LEVEL`

Reply:
Agreed